buildconfig/dev-packages/deb/mantid-developer/ns-control

```diff
@@ -3,15 +3,17 @@ Priority: optional
 Standards-Version: 3.9.2
 
 Package: mantid-developer
-Version: 1.6
+Version: 2.0
 Maintainer: Mantid Project <mantid-tech@mantidproject.org>
 Priority: optional
 Architecture: all
 Depends: git,
   g++,
   clang-format-6.0,
-  cmake-qt-gui(>=3.5),
-  ninja-build,
+  cmake-qt-gui(>=3.13),
```

Comment:
This cmake comes from some other repository right? Is this documented somewhere? I had to scroll back in slack channels to find that...

Reply:
Yep. David added it to the developer instructions: http://developer.mantidproject.org/GettingStarted.html#ubuntu